service/history/ndc/replication_task.go

```diff
@@ -103,10 +103,9 @@ var (
 	ErrLastEventIsNotContinueAsNew = serviceerror.NewInvalidArgument("last event is not continue as new")
 )
 
-func newReplicationTask(
+func newReplicationTaskFromRequest(
 	clusterMetadata cluster.Metadata,
 	historySerializer serialization.Serializer,
-	taskStartTime time.Time,
```

Comment:
can you help me understand why `taskStartTime` is no longer needed?

Reply:
the variable is no longer used by downstream APIs